database/things/f-sharp.pldb

```diff
@@ -7,6 +7,7 @@ reference https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/inde
 aka f#
 aka fsharp
 fileExtensions fs fsi fsx fsscript
+originCommunity Microsoft Research
 domainName fsharp.org
```

Comment:
country Unknown?

Reply:
Don Syme was at MSR in Cambridge. I guess it should be the UK then.